### PR TITLE
fix(selectListItem): fix selected list item

### PR DESCRIPTION
**Pull request by zubaria1**
## Description
fixed selected list item on mobile

## Screenshots (if appropriate):
Screenshot_20210222-172927
Screenshot_20210222-172920

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by AijazBlueeast — APPROVED** [review]
in react native paper there isn't any selected prop so we pass the selected prop and give the list colour on light and dark mode

File: src/components/ListItem/ListItem.tsx [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
 import { List } from 'react-native-paper';
 import React from 'react';
-import { Theme } from '@bluebase/core';
 import { componentMapper } from '@bluebase/component-mapper';
 
+export const selectListItem = (props: any) => {
+	const theme = useTheme();
+	const styles = useStyles('ListItem', props, {
+		background: {
+			backgroundColor: theme.theme.mode === 'dark' ? '#686868' : '#dbdbdb',
```
Comment (artalat) [inline src/components/ListItem/ListItem.tsx:11]: @AijazBlueeast please discuss with me

**Review by artalat — CHANGES_REQUESTED** [review]

**Review by artalat — APPROVED** [review]

**Pull request merged**